src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -3,52 +3,83 @@ pragma solidity 0.8.16;
 
 import "@openzeppelin/contracts/utils/cryptography/ECDSA.sol";
 import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
-import "@arbitrum/nitro-contracts/src/precompiles/ArbSys.sol";
-import "./interfaces/IL1SecurityCouncilUpdateRouter.sol";
 import "./SecurityCouncilMgmtUtils.sol";
 import "@openzeppelin/contracts/utils/Address.sol";
 import "./interfaces/ISecurityCouncilManager.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
 import "../ArbitrumTimelock.sol";
+import "../UpgradeExecutor.sol";
+import "../L1ArbitrumTimelock.sol";
+import "@arbitrum/nitro-contracts/src/precompiles/ArbSys.sol";
+import "./SecurityCouncilUpgradeAction.sol";
 
-/// @notice Manages the security council updates.
-///         Receives election results (replace cohort with 6 new members), add-member actions, and remove-member actions,
-///         and dispatches them to all security councils on all relevant chains
-contract SecurityCouncilManager is Initializable, ArbitrumTimelock, ISecurityCouncilManager {
+contract SecurityCouncilManager is
+    Initializable,
+    AccessControlUpgradeable,
+    ISecurityCouncilManager
+{
     using ECDSA for bytes32;
 
     // cohort arrays are source-of-truth for security council; the maximum 12 owners security council owners should always be equal to the
     // sum of these two arrays (or have pending x-chain messages on their way to updating them)
-    address[] public marchCohort;
-    address[] public septemberCohort;
+    address[] public firstCohort;
+    address[] public secondCohort;
+
+    // address of l1 timelock used for core govervor proposals
+    address public l1CoreGovTimelock;
+    // address of l2 timelock used for core govervor proposals
+    address payable public l2CoreGovTimelock;
```

Comment:
why payable

Reply:
timelock contract is payable, so compiler wants payable when connect to it here:
https://github.com/ArbitrumFoundation/governance/blob/54555c75dc0a0c69950fe8c644b304c1f8490729/src/security-council-mgmt/SecurityCouncilManager.sol#L378